src/main/java/software/amazon/payloadoffloading/PayloadStorageConfiguration.java

```diff
@@ -212,4 +182,18 @@ public boolean isAlwaysThroughS3() {
     public void setAlwaysThroughS3(boolean alwaysThroughS3) {
         this.alwaysThroughS3 = alwaysThroughS3;
     }
+
+    public PayloadStorageConfiguration withServiceSideEncryption(ServerSideEncryptionStrategy serverSideEncryptionStrategy) {
```

Comment:
Can we get some comments on these new methods

Reply:
Done, I've also added a description of the usage in the class docs.